logic/src/commonMain/kotlin/com/wire/kalium/logic/data/notification/LocalNotification.kt

```diff
@@ -45,29 +50,29 @@ sealed class LocalNotificationMessage(open val author: LocalNotificationMessageA
     // shared file, picture, reaction
     data class Comment(
         override val author: LocalNotificationMessageAuthor,
-        override val time: String,
+        override val time: Instant,
         val type: LocalNotificationCommentType
     ) : LocalNotificationMessage(author, time)
 
     data class Knock(
         override val author: LocalNotificationMessageAuthor,
-        override val time: String
+        override val time: Instant
     ) : LocalNotificationMessage(author, time)
 
     data class ConnectionRequest(
         override val author: LocalNotificationMessageAuthor,
-        override val time: String,
+        override val time: Instant,
         val authorId: QualifiedID
     ) : LocalNotificationMessage(author, time)
 
     data class ConversationDeleted(
         override val author: LocalNotificationMessageAuthor,
-        override val time: String
+        override val time: Instant
     ) : LocalNotificationMessage(author, time)
 }
 
-data class LocalNotificationMessageAuthor(val name: String, val imageUri: String?)
+data class LocalNotificationMessageAuthor(val name: String, val imageUri: UserAssetId?)
 
 enum class LocalNotificationCommentType {
-    PICTURE, FILE, REACTION, MISSED_CALL, NOT_SUPPORTED_YET
+    PICTURE, FILE, REACTION, MISSED_CALL, KNOCK, NOT_SUPPORTED_YET
```

Comment:
not sure why KNOCK  is missing in DEV, maybe @borichellow can help me figure out if i am missing something here

Reply:
Because it's connected with ping feature which is not in rc, I prepared another class to easier handle different type of notification with different sound